site/content/en/docs/Guides/troubleshooting.md

```diff
@@ -5,6 +5,206 @@ weight: 200
 description: "Troubleshooting guides and steps."
 ---
 
+## Something went wrong with my GameServer
+
+If there is something going wrong with your GameServer, there are a few approaches to determining the cause:
+
+Depending on what is happening, you may want to check the `kubectl describe <gameserver name>` to view the events
+that are associated with that particular `GameServer` resource. This can give you insight into what the lifecycle of the
+`GameServer` and if anything has gone wrong.
+
+For example:
+
+```shell
```

Comment:
the shell syntax highlighting is making some numbers randomly blue which is strange. Maybe just leave it without a language?